assets/xml/textures/do_action_static.xml

```diff
@@ -0,0 +1,91 @@
+        <Texture Name="gEnterActionLabelTexGER" OutName="enter_action_label_ger" Format="ia4" Width="48" Height="16"/>
+        <Texture Name="gReturnActionLabelTexGER" OutName="return_action_label_ger" Format="ia4" Width="48" Height="16"/>
+        <Texture Name="gOpenActionLabelTexGER" OutName="open_action_label_ger" Format="ia4" Width="48" Height="16"/>
+        <Texture Name="gJumpActionLabelTexGER" OutName="jump_action_label_ger" Format="ia4" Width="48" Height="16"/>
+        <Texture Name="gDecideActionLabelTexGER" OutName="decide_action_label_ger" Format="ia4" Width="48" Height="16"/>
+        <Texture Name="gDiveActionLabelTexGER" OutName="dive_action_label_ger" Format="ia4" Width="48" Height="16"/>
+        <Texture Name="gFasterActionLabelTexGER" OutName="faster_action_label_ger" Format="ia4" Width="48" Height="16"/>
+        <Texture Name="gThrowActionLabelTexGER" OutName="throw_action_label_ger" Format="ia4" Width="48" Height="16"/>
+        <Texture Name="gUnusedNaviActionLabelTexGER" OutName="navi_action_label_unused_ger" Format="i4" Width="48" Height="16"/>
+        <Texture Name="gClimbActionLabelTexGER" OutName="climb_action_label_ger" Format="ia4" Width="48" Height="16"/>
+        <Texture Name="gDropActionLabelTexGER" OutName="drop_action_label_ger" Format="ia4" Width="48" Height="16"/>
+        <Texture Name="gDownActionLabelTexGER" OutName="down_action_label_ger" Format="ia4" Width="48" Height="16"/>
+        <Texture Name="gSaveActionLabelTexGER" OutName="save_action_label_ger" Format="ia4" Width="48" Height="16"/>
+        <Texture Name="gSpeakActionLabelTexGER" OutName="speak_action_label_ger" Format="ia4" Width="48" Height="16"/>
+        <Texture Name="gNextActionLabelTexGER" OutName="next_action_label_ger" Format="ia4" Width="48" Height="16"/>
+        <Texture Name="gGrabActionLabelTexGER" OutName="grab_action_label_ger" Format="ia4" Width="48" Height="16"/>
+        <Texture Name="gStopActionLabelTexGER" OutName="stop_action_label_ger" Format="ia4" Width="48" Height="16"/>
+        <Texture Name="gPutAwayActionLabelTexGER" OutName="put_away_action_label_ger" Format="ia4" Width="48" Height="16"/>
+        <Texture Name="gReelActionLabelTexGER" OutName="reel_action_label_ger" Format="ia4" Width="48" Height="16"/>
+        <Texture Name="g1ActionLabelTexGER" OutName="1_action_label_ger" Format="ia4" Width="48" Height="16"/>
+        <Texture Name="g2ActionLabelTexGER" OutName="2_action_label_ger" Format="ia4" Width="48" Height="16"/>
+        <Texture Name="g3ActionLabelTexGER" OutName="3_action_label_ger" Format="ia4" Width="48" Height="16"/>
+        <Texture Name="g4ActionLabelTexGER" OutName="4_action_label_ger" Format="ia4" Width="48" Height="16"/>
+        <Texture Name="g5ActionLabelTexGER" OutName="5_action_label_ger" Format="ia4" Width="48" Height="16"/>
+        <Texture Name="g6ActionLabelTexGER" OutName="6_action_label_ger" Format="ia4" Width="48" Height="16"/>
```

Comment:
due to the way the indexing of these textures work, the translations are *guaranteed* to be in the same order as their english counterparts. so you should be able to name the german and french ones now. would just use the english word with the language suffix at the end

Reply:
oh i see, you have some unnamed in english as well. Any reason why, id expect you to be able to read it in a texture editor

Reply:
are these the diving numbers actually?
eitherway i think we can call all of these like `gDoAction<Name><Lang>Tex`, since we call these "do actions" in code
and to make this more clear we can call the diving numbers Num1 or something